.pre-commit-hooks.yaml

```diff
@@ -65,3 +65,9 @@
     entry: "\uFFFD"
     language: pygrep
     types: [text]
+-   id: rst-directive-colons
```

Comment:
these should be alphabetically sorted -- can you add a test for this and fix it? (if not, I can add a test -- just fixing it is fine)

Reply:
I've fixed it here, I can add a test for this in a separate PR if you like, have opened #34 to keep track of this